From 9b78adfe70127dba09cf49b7fcc005f921b819bf Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 21 Jul 2026 15:23:11 +0800
Subject: [PATCH 4/4] AOSCOS: quirks: disable touchpad-edge volume/brightness
 sliders on EAECIS NL60R

The touchpad found on the EAECIS NL60R comes with volume and brightness
sliders on the left and right edges, which are extremely easy to trigger
when typing.

Disable them so that we don't get "volume and brightness jitters" when
typing on this laptop.

Co-authored-by: liushuyu <liushuyu011@gmail.com>
Signed-off-by: liushuyu <liushuyu011@gmail.com>
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 quirks/50-system-eaecis.quirks | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 quirks/50-system-eaecis.quirks

diff --git a/quirks/50-system-eaecis.quirks b/quirks/50-system-eaecis.quirks
new file mode 100644
index 00000000..f6000770
--- /dev/null
+++ b/quirks/50-system-eaecis.quirks
@@ -0,0 +1,13 @@
+# Do not edit this file, it will be overwritten on update
+
+# The touchpad found on the EAECIS NL60R comes with volume and brightness
+# sliders on the left and right edges, which are extremely easy to trigger
+# when typing.
+#
+# Disable them so that we don't get "volume and brightness jitters" when
+# typing on this laptop.
+[Disable EAECIS NL60R Touchpad Keys]
+MatchName=PNP0C50:00 36B6:C005 Keyboard
+MatchUdevType=keyboard
+MatchDMIModalias=dmi:*:svnEAECIS:pnNL60R*:*
+AttrEventCode=-EV_KEY:0xe1;-EV_KEY:0xe0;-EV_KEY:0x72;-EV_KEY:0x73;
-- 
2.55.0

